src/components/MainButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Button } from 'react-native';
+
+interface MainButtonProps {
+    title: string;
+  }
+
+const MainButton: React.FC <MainButtonProps> = ({title}) => {
+  return (
+    <>
+      <Button title={title}/>
+    </>
+  );
+};
+
+export default MainButton;
```

Comment:
Eu recomendo evitar export default, porque isso implica que quem for importar esse módulo pode dar qualquer nome no import. Só que nós queremos nomear cada módulo e cada variável nossa de forma clara, e portanto utlilizar essa nomenclatura em todos os lugares que o componente é usado. Faz sentido?

Vale pros seus outros arquivos tbm.